examples/tracer-web/examples/fetchXhr/index.js

```diff
@@ -0,0 +1,100 @@
+import { context, trace } from '@opentelemetry/api';
+import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/sdk-trace-base';
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-http';
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
+import { FetchInstrumentation } from '@opentelemetry/instrumentation-fetch';
+import { XMLHttpRequestInstrumentation } from '@opentelemetry/instrumentation-xml-http-request';
+import { ZoneContextManager } from '@opentelemetry/context-zone';
+import { registerInstrumentations } from '@opentelemetry/instrumentation';
+
+const provider = new WebTracerProvider();
+provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+provider.addSpanProcessor(new SimpleSpanProcessor(new OTLPTraceExporter()));
```

Comment:
Consider to use a BatchSapnProcessor for OTLP exporter as it is more production realistic.

Reply:
Batch processor is more realistic for production but the simple processor is good for examples because data is immediately exported to whatever backend the customer is using and they don't have any configuration or waiting periods. I can go either way here.

Reply:
I'll add a comment for anyone reviewing these examples that they should consider using the BatchSpanProcessor for production